core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Workflow.scala

```diff
@@ -272,12 +272,11 @@ class Workflow(
 
   def getLink(linkID: LinkIdentity): LinkStrategy = idToLink(linkID)
 
-  def getPythonWorkers: Iterable[ActorVirtualIdentity] =
```

Comment:
can we keep this method? it is convenient if I want to send some control messages to all python workers.

Reply:
As I said, the old one is not available anymore because we don't have the `IOperatorExecutor` on the controller side.